src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,29 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, setSelectedTab }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            data-cy="Tab"
+            className={cn({ 'is-active': selectedTab.id === tab.id })}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => setSelectedTab(tab)}
```

Comment:
Consider implementing a check here to invoke the function only if the tab isn't selected